pages/docs/admin-api.mdx

```diff
@@ -1,78 +0,0 @@
----
```

Comment:
@marcklingen I think this page was only available shortly, so I don't think we need a redirect here to the new section. Let me know if you disagree.